clouds/clouds-azure/src/test/resources/azure/armTemplates/eapWebApp.json

```diff
@@ -0,0 +1,103 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "appName": {
+      "type": "string",
+      "defaultValue": "eapAppService"
+    },
+    "linuxFxVersion": {
+      "type": "string",
+      "defaultValue": "JBOSSEAP|7-java11"
```

Comment:
For my knowledge this is actually identifying EAP image?

Reply:
Yes.